modules/rubiconBidAdapter.js

```diff
@@ -368,26 +280,77 @@ function RubiconAdapter() {
           return memo;
         }, {'rpfl_elemid': bidRequest.placementCode});
 
-      try {
-        bidmanager.addBidResponse(bidRequest.placementCode, bid);
-      } catch (err) {
-        utils.logError('Error from addBidResponse', null, err);
-      }
-    });
-    // Run the Emily user sync
+      bids.push(bid);
+
+      return bids;
+    }, []);
+
     hasUserSyncFired = syncEmily(hasUserSyncFired);
```

Comment:
This should be on the spec now as a user sync, right? It hasn't been implemented yet, pending #1549... but I doc'd it.

```
 * @property {function(SyncOptions, Array): UserSync[]} [getUserSyncs] Given an array of all the responses
 *   from the server, determine which user syncs should occur. The argument array will contain every element
 *   which has been sent through to interpretResponse. The order of syncs in this array matters. The most
 *   important ones should come first, since publishers may limit how many are dropped on their page.
```

That should also help smooth over your `resetUserSync()` issue.